ios/NotificationServiceExtension/Info.plist

```diff
@@ -9,5 +9,9 @@
 		<key>NSExtensionPrincipalClass</key>
 		<string>$(PRODUCT_MODULE_NAME).NotificationService</string>
 	</dict>
+	<key>CFBundleVersion</key>
+	<string>1.4.23.0</string>
+	<key>CFBundleShortVersionString</key>
+	<string>1.4.23</string>
```

Comment:
Not sure if adding this in is totally necessary, but make sure these match the main bundle [here](https://github.com/Expensify/App/blob/e372de1fbba49e70f2f79694eb74cd6abefc06a2/ios/NewExpensify/Info.plist#L43) before merging